dev-docs/bidders/criteo.md

```diff
@@ -44,7 +44,7 @@ of this value. See [Issue 6381](https://github.com/prebid/Prebid.js/issues/6381)
 |-------------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|-----------|
 | `minduration`     | optional | Minimum ad duration in seconds                                                                                                                                                                                                                                               | `5`     | `integer` |
 | `startdelay`      | optional | Duration offset (in second) from the start of the content for showing the video ad before the start of the Video. Pre-roll: `0` (default); Mid-roll: `>0`; Default mid-roll: `-1`; Post-roll: `-2`;                                                                          | `5`     | `integer` |
-| `playbackmethod`  | required | Defines how is initiated the video inventory. Page Load with Sound On: `1`; Page Load with Sound Off: `2`; Click with Sound On: `3`; Mouse-Over with Sound On: `4`; Entering Viewport with Sound On: `5`; Entering Viewport with Sound Off by Default: `6`;                  | `1`     | `integer` |
+| `playbackmethod`  | required | Defines how is initiated the video inventory. Page Load with Sound On: `1`; Page Load with Sound Off: `2`; Click with Sound On: `3`; Mouse-Over with Sound On: `4`; Entering Viewport with Sound On: `5`; Entering Viewport with Sound Off by Default: `6`;                  | `[4, 5]`     | `Array<integer>` |
```

Comment:
Just some small edits on the grammar: 

>>Defines how is initiated the video inventory.<<

Should be: 

 Defines how the video inventory is initiated.

Reply:
Indeed, thanks for pointing it out
Just fixed it